components/CreateOrganization.js

```diff
@@ -39,100 +37,130 @@ class CreateOrganization extends React.Component {
       this.setState({
-        result: { error: 'Please accept the terms of fiscal sponsorship' },
+        result: { error: 'Please verify that you are an authorized representative of this organization' },
       });
       return;
     }
 
     this.setState({ status: 'loading' });
-    CollectiveInputType.type = 'ORGANIZATION';
+
+    delete organization.authorization;
 
     try {
-      const res = await this.props.createCollective(CollectiveInputType);
-      const collective = res.data.createCollective;
-      const collectiveUrl = `${window.location.protocol}//${window.location.host}/${collective.slug}?status=collectiveCreated&CollectiveId=${collective.id}`;
-      this.setState({
-        status: 'idle',
-        result: {
-          success: `Organization created successfully: ${collectiveUrl}`,
+      const response = await this.props.createOrganization({
+        variables: {
+          organization,
         },
       });
+
+      if (response) {
+        await this.props.refetchLoggedInUser();
+        const member = await this.props.LoggedInUser.memberOf.filter(
+          member => member.collective.id === response.data.createOrganization.legacyId,
+        );
+        const adminList = this.state.admins.filter(admin => {
+          if (admin.member.id !== this.props.LoggedInUser.collective.id) {
+            return admin;
+          }
+        });
+
+        this.setState({
+          admins: [...adminList, { role: 'ADMIN', member: this.props.LoggedInUser.collective, id: member[0].id }],
+        });
+
+        await this.props.editCollectiveMembers({
+          variables: {
+            collectiveId: response.data.createOrganization.legacyId,
+            members: this.state.admins.map(member => ({
+              id: member.id,
+              role: member.role,
+              member: {
+                id: member.member.id,
+                name: member.member.name,
+              },
+            })),
+          },
+        });
+      }
       await this.props.refetchLoggedInUser();
+
       Router.pushRoute('collective', {
-        CollectiveId: collective.id,
-        slug: collective.slug,
+        slug: response.data.createOrganization.slug,
         status: 'collectiveCreated',
-      });
+      }).then(() => window.scrollTo(0, 0));
     } catch (err) {
       const errorMsg = getErrorFromGraphqlException(err).message;
-      this.setState({ result: { error: errorMsg } });
+      this.setState({ result: { error: errorMsg }, status: 'error' });
       throw new Error(errorMsg);
     }
   }
 
   render() {
     const { LoggedInUser } = this.props;
-
-    const title = 'Create a new organization';
+    const { result, collective, status } = this.state;
 
     return (
-      <div className="CreateOrganization">
-        <Header
-          title={title}
-          className={this.state.status}
-          LoggedInUser={LoggedInUser}
-          menuItems={{ pricing: true, howItWorks: true }}
-        />
-
+      <Container>
```

Comment:
There's a lot of extra `Containers` not doing anything - I think you can remove this one as well. The one on line 112 to wrap around the `LoggedInUser`/`!LoggedInUser` conditions is enough. Also if you need to just wrap some elements but you don't need to use any centering or other properties, you can use `Box` or React's inbuilt `Fragment`